docker/sirius-lpa-frontend/Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM node:21.1.0-alpine3.17 as asset-env
 
 WORKDIR /app
 
+RUN apk add --no-cache git
```

Comment:
Do we need git in this image? I think it should not be included in the production image.

Reply:
Its needed as we run yarn below in this file. I'm unsure what has caused this issue to appear now/recently but without installing git, running `make build` fails on the yarn step with the error: `error Couldn't find the binary git`

Reply:
Since this is a multi-step build, git doesn't end up on the production image (which starts at `FROM alpine:3`), it's only needed when installing the NPM dependencies on the intermediate build step